Makefile

```diff
@@ -12,8 +12,8 @@ include Makefile.options
 ##			      Internals
 
 ## Required binaries
-ELIOMC            := eliomc
-ELIOMOPT          := eliomopt
+ELIOMC            := eliomc ${WARNING_FLAGS}
+ELIOMOPT          := eliomopt ${WARNING_FLAGS}
```

Comment:
Shouldn't we do the same change for `JS_OF_ELIOM`?

Reply:
Effectivement. C'est fait.